src/zen_octet.c

```diff
@@ -2017,7 +2018,7 @@ int luaopen_octet(lua_State *L) {
 		{"charcount", charcount},
 		{"rmchar", remove_char},
 		{"compact_ascii", compact_ascii},
-		{"zcash_topoint", zcash_topoint},
+		// {"zcash_topoint", zcash_topoint},
```

Comment:
Why did we move zcash from zen_octet to zen_ecp/zen_ecp2? 
When I have an octet, I may not be aware of whether it is an ECP or an ECP2, and the algorithm is the same for both. Before I could import the zcash octet and then check its type (it could be both `zenroom.ecp` and `zenroom.ecp2`).
Moreover, it increases code duplication, because we have almost the same function both in zen_ecp and zen_ecp2.

Reply:
If we want also `ECP.from_zcash` that also asserts the type of the decoded object, I would remove the static from zcash_topoint, make it extern in the other file and add a function that calls it and assert the type of the decoded object immediately after.

Reply:
Resons for this:
1. readability: by making explicit returned type, the lua code is much more readable in absence of type checks
2. awareness: who writes lua code is forced to declare ECP type
3. compliance: en/coding functions are all from_/to_ form in zenroom's lua

code duplication may be fixed, but I am not in favor of making conversion types implicit and possibly concealed, as this may lead to mistakes that are very hard to detect in implementations.